doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -977,32 +977,31 @@ The fully expanded example above (without environment variables) looks like this
 
   curl -H "X-Dataverse-key: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" -X DELETE https://demo.dataverse.org/api/datasets/24/versions/:draft
 
-Set Citation Date Field for a Dataset
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Set Citation Date Field Type for a Dataset
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Sets the dataset field type to be used as the citation date for the given dataset (if the dataset does not include the dataset field type, the default logic is used). The name of the dataset field type should be sent in the body of the request.
-To revert to the default logic, use ``:publicationDate`` as the ``$DATASET_FIELD_TYPE_NAME``.
-Note that the dataset field used has to be a date field.
+Sets the dataset citation date field type for a given dataset. ``:publicationDate`` is the default.
+Note that the dataset citation date field type must be a date field.
```

Comment:
Is it worth enumerating a few fields? How would anyone know which fields to use? I see `:dateOfDeposit` below but should we list it and a few other candidates? (I don't know which are popular to use.) Something like...

Examples include:

- :dateOfDeposit
- [ another example ]
- [ another example ]